$DragonFly: src/sbin/dhclient/common/Attic/ctrace.c.patch,v 1.1 2006/01/18 02:33:36 dillon Exp $

Index: contrib/dhcp-3.0/common/ctrace.c
===================================================================
RCS file: /cvs/src/contrib/dhcp-3.0/common/ctrace.c,v
retrieving revision 1.1
diff -u -r1.1 ctrace.c
--- ctrace.c.orig	4 Jan 2005 19:58:53 -0000	1.1
+++ ctrace.c		17 Jan 2006 23:26:03 -0000
@@ -284,7 +284,7 @@
 {
 	u_int32_t *seed;
 
-	if (length != sizeof seed) {
+	if (length != sizeof *seed) {
 		log_error ("trace_seed_input: wrong size (%d)", length);
 	}
 	seed = (u_int32_t *)buf;
